TeensyDmx.cpp

```diff
@@ -1686,6 +2141,58 @@ void TeensyDmx::handleByte(uint8_t c)
             maybeIncrementLengthMismatch();
             m_state = State::IDLE;
             break;
+        case State::RDM_DUB_PREAMBLE:
+            if (c == 0xAA) {
+              // We're not interested in DUB preamble bytes, so don't store them
+              // Start storing at index 8 so we can use DiscUniqueBranchResponse struct
+              m_dmxBufferIndex = 8;
+              m_state = State::RDM_DUB_RECV;
+            } else if (c == 0xFE) {
+              // Discarding bytes
+              // TODO(Peter): Check we only ever get 7 of these
```

Comment:
How much does it matter that there's only 7?  If it can break the controller then we really need to check, if not then it probably doesn't matter.

Reply:
We are the controller! We should probably discard a packet with excess 0xFE's, as there's a chance it could be a collision, so we can discard it early.

Reply:
When I said break the controller, I meant the controller (written here).  However, I understand the checksum will handle this, so it shouldn't be an issue.